src/replication.c

```diff
@@ -2993,109 +2994,97 @@ void dualChannelSyncHandleRdbLoadCompletion(void) {
     return;
 }
 
-/* Try a partial resynchronization with the primary if we are about to reconnect.
- * If there is no cached primary structure, at least try to issue a
- * "PSYNC ? -1" command in order to trigger a full resync using the PSYNC
- * command in order to obtain the primary replid and the primary replication
- * global offset.
+/**
+ * Handles the initial step of the partial resynchronization process by
```

Comment:
nit - I don't remember we start the comment block with `/**`?
```suggestion
/* Handles the initial step of the partial resynchronization process by
```